test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua

```diff
@@ -0,0 +1,147 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     SDL receives request for app activation from HMI and LocalPT contains revoked permission for the named application
+--     1. Used preconditions:
+--			Delete SDL log file and policy table
+--			Close current connection
+--			Make backup copy of preloaded PT
+--			Overwrite preloaded PT adding list of groups for specific app
+--			Connect device
+--			Register app
+--   		Revoke app group by PTU
+-- 			
+--     2. Performed steps
+-- 			Activate app
+--
+-- Requirement summary: 
+--    [Policies] SDL.ActivateApp from HMI and 'isAppPermissionsRevoked' parameter in the response
+--
+-- Expected result:
+--      PoliciesManager must respond with "isAppPermissionRevoked:true" and "AppRevokedPermissions" param containing the list of revoked permissions to HMI
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local Preconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+function Test:CloseConnection()
```

Comment:
Please rename functions in this section to contain Precondition_ e.g. 
function Test:Precondition_CloseConnection()